sdk/src/trace/span.h

```diff
@@ -42,7 +42,8 @@ class Span final : public trace_api::Span
 
   bool IsRecording() const noexcept override;
 
-  trace_api::Tracer &tracer() const noexcept override { return *tracer_; }
+  trace_api::SpanContext GetContext() const noexcept override { return trace_api::SpanContext(); }
```

Comment:
Why is this returning a default constructed `SpanContext`? Shouldn't `Span` store a `SpanContext` that it returns when you call `GetContext()`?

Reply:
I think it doesn't matter because this class seems to be incompletely defined and it depends on other Children class that defines the behavior of it. And often times we just want to behave simply by returning a invalid spancontext while not risking to confine people on a must-have span_context_field (it could be a span that contains a list of other spans, and will use one of them at random for spancontext)

Reply:
Would it make sense to have a `span_context_` member in this class? It may be needed for propagating IDs

Reply:
I've added the span_context_ field to span.h